SignalMessaging/utils/UIUtil.m

```diff
@@ -33,6 +33,14 @@ + (void)setupSignalAppearence
 
     UIBarButtonItem.appearance.tintColor = Theme.navbarIconColor;
 
+    // Using the keyboardAppearance causes crashes due to a bug in UIKit.
```

Comment:
There appears to be a bug in UIKit.  It's known on StackOverflow.

Reply:
Can you link to any relevant URL or leave any other breadcrumb about how this might ever be resolved?

Reply:
It's very reproducible. Others have reported seeing the same issue on StackOverflow.  However, I have found nothing worth linking to.